WcaOnRails/config/locales/de.yml

```diff
@@ -1010,6 +1032,29 @@ de:
           Eventkombinationen verbieten willst. Bitte fülle dies in Englisch aus!
         #original_hash: 1067798
         main_event_id: Der Gewinner dieses Events ist der Gewinner der Competition.
+        #original_hash: da39a3e
+        allow_registration_without_qualification: ''
+        championships:
+          #original_hash: da39a3e
+          championship_type: ''
+        competition_series:
+          #original_hash: da39a3e
+          wcif_id: ''
+          #original_hash: 77811b5
+          name: >-
+            Der volle Name der Series Competition, inklusive des Jahres am Ende
+            (z.B. Southern Australia 2022). Achte auf Großschreibung. Dieselben
+            Regeln wie für Namen von individuellen Wettbewerben finden auch hier
+            Anwendung.
+          #original_hash: 778c20c
+          short_name: >-
+            Ein Kurzname zum Anzeigen. Dieselben Regeln wie für den Namen des
+            individuellen Wettbewerbs finden auch hier Anwendung.
+          #original_hash: 3e27501
+          competition_ids: >-
+            <b>Dieses Feld kann nicht editiert werden!</b><br/>Um Wettbewerbe
+            dieser Serie hinzuzufügen, musst zur individuellen 'Bearbeite'-Seite
```

Comment:
When I was coding this feature, I had the German translation "zur _jeweiligen_ 'Bearbeite'-Seite" in the back of my head. Although arguably either translation is fine

Reply:
I see, I'm also fine with _jeweiligen_, and I missed a "du" anyways...